genesyscloud/telephony_providers_edges_phonebasesettings/genesyscloud_telephony_providers_edges_phonebasesettings_schema.go

```diff
@@ -68,9 +68,28 @@ var (
 			},
 		},
 	}
-)
 
-func ResourcePhoneBaseSettings() *schema.Resource {
+	lineBase := &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"line_meta_base_id": {
+				Description: "lineMetaBaseId is computed by the provider based on the phoneBaseSettings using the phoneBaseSetting's template",
+				Type:        schema.TypeString,
+				Optional:    true,
```

Comment:
Optional and Required should be set to `false` if this is truly a computed only attribute.

Reply:
it cannot be read only, because we have some users who export the resources from one org and install them to another, in those cases the provider will force them to remove that field.